rust/cubestore/cubestore/src/metastore/rocks_store.rs

```diff
@@ -331,3 +350,470 @@ macro_rules! meta_store_table_impl {
         }
     };
 }
+
+pub trait RocksStoreDetails: Send + Sync {
+    fn open_db(&self, path: &Path) -> Result<DB, CubeError>;
```

Comment:
I've changed `impl AsRef<Path>` to &Path, because it's impossible to use it in dynamic dispatching due to Rust restrictions. It's a breaking change